frontend/src/pages/AnnouncementPage.js

```diff
@@ -0,0 +1,330 @@
+// SPDX-License-Identifier: MIT
+// SDPX-FileCopyrightText: 2021 Que Le <b.le@tu-berlin.de>
+
+import React, {
+	useEffect,
+	useState
+} from "react";
+import appwriteApi from "../api/appwriteApi";
+
+import Grid from "@mui/material/Grid";
+import { makeStyles } from "@mui/styles";
+import {
+	Button, Alert,
+	Table, TableBody,
+	TableCell, TableRow,
+	TextField, Typography,
+} from "@mui/material";
+import Box from "@mui/material/Box";
+import Container from "@mui/material/Container";
+import Collapse from "@mui/material/Collapse";
+
+function InputFields({ defaultTitle, defaultContent, titleComponenId, contentComponenId }) {
+	const useStyles = makeStyles(() => ({
+		input: {
+			color: "#FFF",
+		},
+	}));
+
+	const classes = useStyles();
+	return <Grid container spacing={2}>
+		<Grid item xs={12}>
+			<TextField
+				required
+				fullWidth
+				name="title"
+				label="Title"
+				id={titleComponenId}
+				color="warning"
+				defaultValue={defaultTitle}
+				inputProps={{ className: classes.input }}
+			/>
+		</Grid>
+		<Grid item xs={12}>
+			<TextField
+				required
+				fullWidth
+				name="content"
+				label="Content"
+				id={contentComponenId}
+				defaultValue={defaultContent}
+				inputProps={{ className: classes.input }}
+			/>
+		</Grid>
```

Comment:
This should not be a single line input field, but should be a scrollable, multi-line input field. As it is now, it is pretty much unusable for texts longer than a sentence.

Reply:
> On another note, the database init script creates all the example documents without any read permissions and we should probably change that soon. We only would need to add `read: ["*"]` to the `create_document` arguments in the init script

It had already:
```python
# backend/database-collection-schemas/createAnnouncementCollection.py
createCollectionResult = database.create_collection(
    "Announcements",  # Collection Name
    ["*"],  # Read permissions
    ["team:Admins"],  # Write permissions
    [
        {
            "label": "created_at",
            "key": "created_at",
            "type": "numeric",
            "required": True,
            "array": False,
        },
        {
            "label": "updated_at",
            "key": "updated_at",
            "type": "numeric",
            "required": False,
            "array": False,
        },
        {
            "label": "title",
            "key": "title",
            "type": "text",
            "required": True,
            "array": False,
        },
        {
            "label": "content",
            "key": "content",
            "type": "text",
            "required": True,
            "array": False,
        },
    ],
)
```

Reply:
This is not enough. That is only to have access to the whole table, but you need to set the permissions on a per document level as well.